### DISCUSSION TITLE
Welcome to cajal Discussions!

### DISCUSSION THREAD

**Discussion by minhajh** [minhajh/cajal#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by VeraSneijders** [comment]
Hi, I'm Vera and a master student of Biomedical Engineering and I'm currently working on my graduation project about vagus nerve stimulation. I'm using Cajal to simulate blocking and was wondering if it is also possible to plot the sodium and potassium current curves without having to insert the Hodgkin-Huxley 'hh' model on all nodes?

**Reply by minhajh** [reply]
Hi Vera, I've replied in a separate discussion (https://github.com/wmglab-duke/cajal/discussions/5).

**Comment by VeraSneijders** [comment]
Hi Minhaj, another question. Instead of an extracellular stimulus I would like to simulate an intracellular stimulation but I couldn't find out how to perform this. Can you help me?

**Reply by minhajh** [reply]
Hi Vera, I've replied to this in a new discussion (https://github.com/wmglab-duke/cajal/discussions/7)
